### ISSUE TITLE
room preview bar [object Object]

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#4052]
![image](https://cloud.githubusercontent.com/assets/2403652/26521403/a5851640-42df-11e7-853e-b2ea51c0cc4f.png)
on /develop/

on /app/ the text is
```
You are trying to access Home Assistant.
Click here to join the discussion!
```

**Comment by uhoreg** [comment]
@MTRNord is this caused by the translation changes?

**Comment by ara4n** [comment]
fixed in matrix-org/matrix-react-sdk@957aa04
